core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/stack/loadbalancer/handler/CreateLoadBalancerEntityHandler.java

```diff
@@ -42,6 +40,8 @@
 import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
 import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
 
+import reactor.bus.Event;
```

Comment:
As I checked only an additional space has been added to the code, why is this import needed?